scripts/src/owners/owners_file.py

```diff
@@ -25,3 +25,26 @@ def get_provider_delivery(owner_data):
         pass
     return provider_delivery
 
+def getFileContent(changed_file):
```

Comment:
This is not quit how I envisaged this.
- the code loads the yaml in two places now, make a load from file method and have two existing cases call it.
- I would then expect a separate method to get each attribute. Calling code then just use the get method which return null if it does exist.  
- Getting the vendor type if not an owners file function so would not expect that part in this file but as part of the caller. 

For info I was extending the owners_file.py for the signed chart work and finished with this which you may use:
```
def get_owner_data(category, organization, chart):
    return get_owner_data_from_file(os.path.join("charts", category, organization, chart, "OWNERS"))

def get_owner_data_from_file(owner_path):
    try:
        with open(owner_path) as owner_data:
            owner_content = yaml.load(owner_data,Loader=Loader)
        return True,owner_content
    except Exception as err:
        print(f"Exception loading OWNERS file: {err}")
        return False,""
```

Reply:
the reason i did not use the get_owner_data function in owners_file.py is because we are getting the file paths from the action itself. So i do not need to pass in category,organization and chart as params to the get_owner_data and hence added the entire method. Let me know if it is fine.